client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -86,8 +86,12 @@
 import java.util.Locale;
 import java.util.StringJoiner;
 
-final class RequestConverters {
-    static final XContentType REQUEST_BODY_CONTENT_TYPE = XContentType.JSON;
+/**
+ * A set of methods used to facilitate conversion of specific actions requests into generic {@link Request}. These are meant to be helper
+ * methods used by subclients, which need to send generic {@link Request}'s down to the {@link RestClient}.
+ */
+public final class RequestConverters {
```

Comment:
I'm torn here because extensions could really use these. But I'm not sure we're really ready to declare them stable. Basically the question is, "should we have to leave a breaking change doc every time we change one of these?"